dimens.xml

```diff
@@ -834,32 +891,36 @@
     <dimen name="qs_footer_dialog_icon_margin">8.0sp</dimen>
     <dimen name="qs_footer_dialog_icon_size">24.0sp</dimen>
     <dimen name="qs_footer_dialog_subtitle_padding">20.0dip</dimen>
-    <dimen name="qs_footer_height">88.0dip</dimen>
+    <dimen name="qs_footer_height">48.0dip</dimen>
     <dimen name="qs_footer_icon_size">16.0dip</dimen>
     <dimen name="qs_footer_padding_end">24.0dip</dimen>
     <dimen name="qs_footer_padding_start">16.0dip</dimen>
-    <dimen name="qs_gutter_height">0.0dip</dimen>
     <dimen name="qs_header_gear_translation">16.0dip</dimen>
-    <dimen name="qs_header_upper_height">45.0dip</dimen>
-    <dimen name="qs_lower_pager_height">104.0dip</dimen>
+    <dimen name="qs_header_margin_top">5.0dip</dimen>
+    <dimen name="qs_header_margin_top_custom">10.0dip</dimen>
+    <dimen name="qs_header_system_icons_area_height">40.0dip</dimen>
+    <dimen name="qs_header_upper_height">62.0dip</dimen>
+    <dimen name="qs_header_upper_height_custom">78.0dip</dimen>
+    <dimen name="qs_lower_pager_height">160.0dip</dimen>

```

Comment:
i changed this line value as per old OOS 5.0.4 (8.0.0) and it changed the padding to normal like 8.0.0 but it was overlapping the date with the qs tiles

@ungeeked